packages/components/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@kaizen/components",
+  "version": "1.0.0",
+  "description": "Kaizen component library",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "future",
+    "src",
+    "dist"
```

Comment:
We'll probably need to double check this. Does this include the source code for stories and tests? I wonder if consumer config would yell at it 🤔

Reply:
Yep, should be able to: `import { FilterDRP } from "@kaizen/components/src/FilterDRP"`

Reply:
Well they shouldn't really exist in a distributed package usually right? I'm more worried about it making their CIs explode with unnecessary errors is all.

Reply:
Fair enough!